test/util.go

```diff
@@ -4,20 +4,53 @@ import (
 	"context"
 	"fmt"
 	"strings"
+	"time"
 
 	testcluster "github.com/mesosphere/ksphere-testing-framework/pkg/cluster"
 	networkutils "github.com/mesosphere/ksphere-testing-framework/pkg/utils/networking"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// timeTimeout defines the maximum time to wait within portForwardPodWithPrefix for a pod and its containers
+	// to be Running and Ready. If we hit the timeout, there are most likely issues with the addon itself.
+	timeTimeout = 300 * time.Second
```

Comment:
```suggestion
	podReadyTimeout = 300 * time.Second
```
`timeTimeout` doesn't really tell what that value is used for.

Reply:
👍